### PR TITLE
Added many more welcome messages.

### LINKED ISSUE #85: Add more unique welcome messages to cycle through
## Feature Reasoning

There can always be more welcome messages to cycle from.\
The current list of 12 starts to repeat itself not after long.

## Feature Description

Add a variety of new welcome messages for the plugin to cycle from.\
The goal for the v.1.2.0 release is to have at least 30 unique welcome messages.

## Additional Context

Current set of welcome messages:
welcome_text

### PR DESCRIPTION
**Pull request by dcechano**
## Associated Issue

Closes #85 

## Testing Details

Operating System: Windows 10
RuneLite Version: 1.7.24

### CODE REVIEW

File: src/main/java/com/hydratereminder/HydrateReminderPlugin.java [hunk 1]
```diff
@@ -123,6 +121,24 @@
...
+						add("Welcome Traveller. Nothing hurts morale like dehydration! Remember to drink water.");
+						add("People who don't believe in magic have obviously never had water!");
+						add("You're 70% water. Don't forget to stay hydrated!");
+						add("\"Thousands have lived without love, not one without water.\" - W.J. Auden");
+						add("Having trouble focusing? Sounds like you need to drink some water ;)");
+						add("\"If there is magic on this planet, it is contained in water.\" - Loren Eiseley");
+						add("Feeling low on energy? Drink some water!");
+						add("Nothing like RuneScape and a tall glass of ice cold water!");
+						add("I suppose we'll allow tea... For now.");
+						add("Type \"::hydrate next\" to set a reminder to stay hydrated!");
```
Comment (jmakhack) [inline src/main/java/com/hydratereminder/HydrateReminderPlugin.java:133]: The next command doesn't exactly set a reminder. Can you update this message to something like:
`Type \"::hydrate next\" to view the time remaining until the next hydration break!`

File: src/main/java/com/hydratereminder/HydrateReminderPlugin.java [hunk 2]
```diff
@@ -123,6 +121,24 @@
...
+						add("Save key strokes by using the short hand \"::hr <command>\" instead of \"::hydrate <command>\"");
+						add("Imagine getting in a sword fight while dehydrated. What. A. Nightmare.");
+						add("Don't forget to stay hydrated while out and about!");
+						add("A wise traveler is a hydrated traveler.");
+						add("Welcome Traveller. Nothing hurts morale like dehydration! Remember to drink water.");
```
Comment (jmakhack) [inline src/main/java/com/hydratereminder/HydrateReminderPlugin.java:138]: Looks like this message is a duplicate of Line 124. Also, `Traveller` should be spelled `Traveler`.

**Review by jmakhack — CHANGES_REQUESTED** [review]
Thanks for contributing all these new messages!
Just left a few comments after reviewing the changes.

**Comment by jmakhack** [comment]
@all-contributors add @dcechano for code

File: src/main/java/com/hydratereminder/HydrateReminderPlugin.java (hunk 2 above)
Comment (dcechano) [reply src/main/java/com/hydratereminder/HydrateReminderPlugin.java:138]: That's embarrassing! I apologize. Should have double checked before submitting. I will fix the typo and duplicate and improve the message you suggested above as well.

**Review by jmakhack — APPROVED** [review]

**Pull request merged**